packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -117,13 +117,13 @@ async fn _run(
     outbound_sender: UnboundedSender<OutboundFromRunnerMsg>,
 ) -> WorkerResult<()> {
     // Open sockets for Python process to connect to (i.e. start listening).
-    let mut nng_sender = NngSender::new(init_msg.experiment_id.clone(), init_msg.worker_index)?;
-    let mut nng_receiver = NngReceiver::new(init_msg.experiment_id.clone(), init_msg.worker_index)?;
+    let mut nng_sender = NngSender::new(init_msg.experiment_id, init_msg.worker_index)?;
+    let mut nng_receiver = NngReceiver::new(init_msg.experiment_id, init_msg.worker_index)?;
 
     // Spawn Python process.
     let mut cmd = Command::new("sh");
     cmd.arg("./src/worker/runner/python/run.sh")
-        .arg(&init_msg.experiment_id)
+        .arg(&init_msg.experiment_id.to_string())
```

Comment:
Why we need to convert it to a `String`? Can't we just borrow it?

Reply:
Because the ID is a `Uuid`